charts/iter8lib/templates/_k-result-secret.tpl

```diff
@@ -0,0 +1,14 @@
+{{- define "k.result.secret" -}}
```

Comment:
I am still wondering about the implications of a single spec and single result secret. This means there is no history of experiments. In practice, only 1 experiment per "group". Is this acceptable? I suppose it the interest of keeping it simple, it might be. Does this impact `delete`?

Reply:
> In practice, only 1 experiment per "group". Is this acceptable?

For now, we definitely want to keep things simple and be able to make progress. This approach simplifies not only `iter8 k` based experimentation, but also simplifies `autox` which we will want to get into later on.

Incidentally, the previous implementation also did not retain history ... it merely named the experiment secrets uniquely with revision numbers, which has the effect of complicating RBACs (different RBACs in each experiment revision).

This does not change `delete`, as delete was always intended to wipe out the entire group (i.e., the underlying Helm release), which it still does.

***

Obviously, if maintaining history becomes important for some new use-case, we can always dump the results of experiments (including Kubernetes experiments someplace, and view it and create reports out of it offline). This possibility still exists.